app/config.py

```diff
@@ -1,14 +1,14 @@
 
 LCS_BASE_URL = ""
- 
+
 DIRECTOR_CREDENTIALS = {
     "email":  "",
     "password": ""
 }
 
 
 # uri should contain auth and default database(database name)
-DB_URI = ""
+DB_URI = "mongodb://127.0.0.1:27017/hackrd"
```

Comment:
I can make a .gitignore for this page so we don't expose any credentials.

Reply:
We should implement a `config.example.py` like [LCS](https://github.com/HackRU/lcs/blob/master/config.example.py), so that each developer stores their own copy of the `config.py` file locally, and the repo only maintains a template file.

Reply:
Can we remove config.py so that github doesn't track it

Reply:
Yup let's remove this, you already added in `config.example.py` so we're good to go!